app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,12 @@
+<nav class="navbar navbar-default">
+  <div class="container-fluid">
+    <div class="navbar-header">
+      <a class="navbar-brand" href="#">Job Tracker</a>
+    </div>
+      <ul class="nav navbar-nav navbar-right">
+        <li><a href="/companies">Companies</a></li>
```

Comment:
leverage your rails helper methods here and use `link_to` and your path helpers!